components/schemas/stacks/builds/StackBuildDeploymentUpdates.yml

```diff
@@ -0,0 +1,26 @@
+title: StackBuildDeploymentUpdates
+type: object
+description: Optional update object used to specify specific params to update from the stack build.
+required:
+  - containers
+  - scoped_variables
+properties:
+  containers:
+    type: object
+    description: A map of the container names to update within the environment.
+    additionalProperties:
+      $ref: "../StackDeployContainersObject.yml"
+    minProperties: 1
+  scoped_variables:
```

Comment:
this is nullable
![image](https://github.com/cycleplatform/api-spec/assets/5110855/06d851a4-5213-4320-987f-d4257d59760f)